readme/installation.md

```diff
@@ -0,0 +1,107 @@
+# Installation
+
+Below are installation instructions for certain distributions as well as
+<a href="#manual-installation">Manual installation</a> instructions.
+
+We welcome contributors to maintain missing distributions.
+
+Please note that distributions may lag with the latest bug fixes.  To
+get the latest bug fixes, use the <a href="#manual-installation">Manual
+installation</a> method.
+
+## Manjaro/Arch
+
+```bash
+yay -S input-remapper-git
+sudo systemctl restart input-remapper
+sudo systemctl enable input-remapper
+```
+
+Please note that the above version may lag from the latest version.  See
+the <a href="#manual-installation">Manual installation</a> method to
+install the latest version.
+
+## Ubuntu/Debian
+
+To get the latest version, use the following method:
+
+```bash
+sudo apt install git python3-setuptools gettext
+git clone https://github.com/sezanzeb/input-remapper.git
+cd input-remapper && ./scripts/build.sh
+sudo apt install -f ./dist/input-remapper-2.0.0.deb
+```
+
+The next two methods may result in the installed version not having the
+latest bug fixes.
+
+### Install using a .deb file
+
+1. Download the `.deb` file from the
+[release page](https://github.com/sezanzeb/input-remapper/releases)
+2. Manually install it
```

Comment:
Add the appropriate command for step 2, otherwise this is not any help for people without debian experience.

`sudo apt install -f /path/to/input-remapper-2.0.0.deb`